src/app/navbar.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import React from "react";
+import Image from "next/image";
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+import styles from "./Navbar.module.css";
+
+export default function Navbar() {
+  const pathname = usePathname();
+
+  const pageLinksData = [
```

Comment:
[Please Change]: Thought I left a comment here on the first review but maybe it didn't save. These constants should be moved outside the component; it makes it clearer that they're supposed to be constants and they won't be re-evaluated each time the component renders.